### PR TITLE
Fix: remove misplaced `.id`

### PR DESCRIPTION
**Pull request by sillyElvy**
## About

This pull request is about (X), which does (Y).

## Checklist

- [ ] I've ran `pre-commit` to format and lint the change(s) made.
- [ ] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- [ ] I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [x] Bugfix

### CODE REVIEW

File: interactions/base.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "__authors__",
 )
 
-__version__ = "4.2.1"
+__version__ = "4.3.0-beta.1"
```
Comment (Toricane) [inline interactions/base.py]: It's already in unstable..?

**Review by Toricane — CHANGES_REQUESTED** [review]
Version

File: interactions/base.py (hunk 1 above)
Comment (sillyElvy) [reply interactions/base.py]: I have no idea..

Comment (Toricane) [reply interactions/base.py]: It is, so why is this showing up? Idk maybe revert that commit and fetch upstream

Comment (sillyElvy) [reply interactions/base.py]: fixed

**Review by Toricane — APPROVED** [review]
LGTM!

**Pull request merged**